app/src/main/java/com/amaze/filemanager/asynchronous/asynctasks/SearchAsyncTask.java

```diff
@@ -123,7 +124,7 @@ private void search(HybridFile directory, final SearchFilter filter) {
           activity.get(),
           rootMode,
           file -> {
-            if (!isCancelled()) {
+            if (!isCancelled() && !(file.getName().startsWith("."))) {
```

Comment:
Isnt there a HybridFile function to check if file is hidden?

Reply:
Oh! Yes. I didn't notice that. Shall do it that way?